### PR TITLE
57 us set up mkdocs

### PR DESCRIPTION
**Pull request by MariaR001**
Follow the following steps (they are also in the readme):

1. Open a terminal & make sure docker is running
2. cd documentation
3. run docker pull polinux/mkdocs
4. docker build -t polinux/mkdocs .
5. docker run -it -p 8080:8000  polinux/mkdocs
6. You should be able to see the mkdocs website: http://localhost:8080/ 
** you can go through docker to open it (don't use http://0.0.0.0:8000/)

If you want to add pages, they should go under documentation/docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,3 +206,17 @@ To use Postman for testing the PostgREST API corrrectly accessing the database:
 ### Adding new configuration properties
 To add a new configuration property to the database, simply add an entry for it in `appconfig_properties.csv` (under ./dataFiles). Then, simply refresh the database using the `refresh_database.sh` script.
 
+## MkDocs Set up with docker
+
+Follow the following steps:
+
+1. Open a terminal & make sure docker is running
+2. cd documentation
+3. run docker pull polinux/mkdocs
```
Comment (lyubageorgieva) [inline README.md:215]: Can we maybe add all the commands to run in bash script so it's easier to see, copy and run?

**Review by lyubageorgieva — CHANGES_REQUESTED** [review]
Except my one comment everything looks good! It runs perfectly!

File: README.md (hunk 1 above)
Comment (MariaR001) [reply README.md:215]: Added!

**Review by lyubageorgieva — APPROVED** [review]
It's perfect!

**Pull request merged**